src/resourceExplorers/LocalResourceExplorer.ts

```diff
@@ -130,17 +125,19 @@ export class LocalResourceExplorer {
     return parseInt(port, 10);
   }
 
-  private async getDescription(port: number, options?: TLocalProjectOptions, description?: string) {
+  private async getDescription(port: number, options?: TLocalProjectOptions) {
     const blockNumber: string | undefined = options?.blockNumber?.Equals(0)
-      ? Constants.defaultLastBlockDescription
+      ? Constants.latestBlock
```

Comment:
generally I'm ok with the code, I just feel constant use of null optional operators is not great longer term. If you have a forked instance, I would hope most of that info would be required/filled in as part of the creation. We just create bugs later on in refactor/new people working on the code after us.

Reply:
I agree :)
In this specific case, I didn't know much about the code, so I preferred to leave the option as an option :P. Anyway, I did a revamp and now all the parameters are required :)